INSTALL.md

```diff
@@ -1,23 +1,114 @@
 # Contents
-- [Contents](#contents)
-  - [Basic Configuration](#basic-configuration)
-  - [Adding Applications](#adding-applications)
-  - [Adding Kratos to Path](#adding-kratos-to-path)
-  - [Examples](#examples)
-    - [Linux](#linux)
-    - [Windows](#windows)
-    - [MacOS](#macos)
-  - [Advanced Configuration](#advanced-configuration)
-    - [Building Environment](#building-environment)
-    - [Common Flags](#common-flags)
-    - [Compilation Performance](#compilation-performance)
-    - [Parallelism](#parallelism)
-    - [Logging](#logging)
-    - [External libraries](#external-libraries)
-      - [Metis](#metis)
-      - [Trilinos](#trilinos)
-  - [Applications](#applications)
+* [Cloning Kratos](#cloning-kratos)
+* [Kratos Dependencies](#kratos-dependencies)
+  * [Kratos Core Dependencies](#kratos-core-dependencies)
+    * [Linux Installation](#linux-installation)
+    * [Windows Installation](#windows-installation)
+  * [Specific Application Dependencies](#specific-application-dependencies)
+* [Basic Configuration](#basic-configuration)
+* [Adding Applications](#adding-applications)
+* [Adding Kratos to Path](#adding-kratos-to-path)
+* [Examples](#examples)
+  * [Linux](#linux)
+  * [Windows](#windows)
+  * [MacOS](#macos)
+* [Advanced Configuration](#advanced-configuration)
+  * [Building Environment](#building-environments)
+  * [Common Flags](#common-flags)
+  * [Compilation Performance](#compilation-performance)
+  * [Parallelism](#parallelism)
+  * [External Libraries](#external-libraries)
+    * [Feast](#feast)
+    * [Metis](#metis)
+    * [Trilinos](#trilinos)
+
+## Cloning Kratos
+
+In order to obtain the source code of Kratos, you will need to clone the repository using git.
+
+You can install git through the following command in Linux:
+```Shell
+sudo apt-get install git
+```
+In Windows, you can download it in:
+
+* [Download Git](https://git-scm.com/downloads)
+
+
+
+Once git is installed you can fetch the code by using this command in a terminal:
+
+```Shell
+git clone https://github.com/KratosMultiphysics/Kratos Kratos
+```
+
+## Kratos Dependencies
+
+### Kratos Core Dependencies
+  This are the basic dependecies needed to compile the Kratos Core and most of the Kratos applications.
+  * Python3-dev
+  * C++11 compiler
+  * CMake
+  * Boost
+
+Additionaly Visual Studio or MinGW is required to compile in Windows.
+
+- #### Linux installation
+
+    The command below will install all the packages needed.
+
+    ```Shell
+    sudo apt-get install python3-dev gcc g++ cmake libboost-all-dev
+    ```
+    Newer versions of boost can be downloaded in:
+
+    http://www.boost.org/users/download/.
+
+- #### Windows installation
+
+    - Visual Studio
+
+        *Visual Studio* is the only compiler officially supported to build *Kratos* under *Windows*.
+        Since the adoption of **C++11** we support versions 2015 update 3 onwards. We recommend you to use the latest version of visual studio which can be obtained here for free:
```

Comment:
@roigcarlo are this versions (C++11 and VS2015) up to date?

Reply:
no, should be at least 2017, but we recommend 2019